frontend/src/context/ListProvider.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useEffect, useCallback, useState } from "react";
+import { ExplorerApi } from "../libraries/explorer-wamp/index";
+
+const ListContext = createContext({
+  transactions: [],
+  blocks: [],
+});
+
+export default (props) => {
+  const [transactions, dispatchTransactions] = useState([]);
+  const [blocks, dispatchBlocks] = useState([]);
+
+  const fetchList = function (stats) {
+    let transactions = stats[0].transactions;
+    let blocks = stats[0].blocks;
+
+    dispatchBlocks(blocks);
+    dispatchTransactions(transactions);
+  };
+
+  const Subscription = useCallback(() => {
+    new ExplorerApi().subscribe("database-list", fetchList);
```

Comment:
We need to unsubscribe from the topic when the provider gets unmounted, otherwise, it will keep receiving the traffic.